.github/workflows/push_pr.yml

```diff
@@ -103,3 +135,12 @@ jobs:
           docker pull nginx
       - name: Run e2e tests
         run: make test-integration
+      - name: Create image for chart testing
```

Comment:
note that since there is no e2e tests I added a step to install and upgrade charts